src/ResultWriter/WaveFieldWriter.h

```diff
@@ -202,7 +202,7 @@ class WaveFieldWriter : private async::Module<WaveFieldWriterExecutor, WaveField
 	 * @param timeTolerance The tolerance in the time for ignoring duplicate time steps
 	 */
 	void init(unsigned int numVars, int order, int numAlignedDOF,
-			const MeshReader &meshReader,
+			const MeshReader &meshReader,  const unsigned *ClusteringFieldMap,
```

Comment:
It's probably more idiomatic to pass a const ref to the std::vector instead.

Reply:
Done